cpp/cmake/thirdparty/get_cccl.cmake

```diff
@@ -36,6 +36,12 @@ function(find_and_configure_cccl)
   # Find or install CCCL with our custom set of patches
   rapids_cpm_cccl(BUILD_EXPORT_SET cudf-exports INSTALL_EXPORT_SET cudf-exports)
 
+  # Remove this after upgrading to a CCCL that has a proper CMake option. See
+  # https://github.com/NVIDIA/cccl/pull/2844
+  target_compile_definitions(
+    cudf PRIVATE THRUST_FORCE_64_BIT_OFFSET_TYPE=0 THRUST_FORCE_32_BIT_OFFSET_TYPE=1
+  )
+
```

Comment:
Sorry I didn't check where you put this. This needs to go into cpp/CMakeLists.txt after the cudf target has been created.

Reply:
Fixed in 816fc09.